packages/wrangler/src/dev.tsx

```diff
@@ -1181,6 +1184,18 @@ export async function validateDevServerSettings(
 	args: StartDevOptions,
 	config: Config
 ) {
+	if (
+		(args.legacyAssets ||
+			config.legacy_assets ||
+			args.experimentalAssets ||
+			config.experimental_assets) &&
+		(args.site || config.site)
+	) {
+		throw new UserError(
```

Comment:
Do we need a similar error preventing `legacyAssets` to be used in conjunction with `experimentalAssets`?

Reply:
we totally do, and I was totes convinced we had that in place 😅 . Thx for catching this! Fixed ✅